prediction_market_agent/utils.py

```diff
@@ -8,6 +8,18 @@
     should_not_happen,
 )
 from pydantic import SecretStr
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class DBKeys(BaseSettings):
+    model_config = SettingsConfigDict(
+        env_file=".env", env_file_encoding="utf-8", extra="ignore"
+    )
+    SQLALCHEMY_DB_URL: t.Optional[str] = None
+
+    @property
+    def sqlalchemy_db_url(self) -> str | None:
+        return self.SQLALCHEMY_DB_URL
```

Comment:
Is this property required if it only renames `SQLALCHEMY_DB_URL` to `sqlalchemy_db_url`?

Reply:
Removed